### PR TITLE
Restore --format 'table...' to commands

### PR DESCRIPTION
**Pull request by jwhonce**
* podman xyz inspect

* Add regex for mapping spaces in {{ *.Id *}} -> {{.ID}}

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: cmd/podman/inspect/inspect.go [hunk 1]
```diff
@@ -3,11 +3,15 @@ package inspect
 import (
 	"context"
 	"fmt"
+	"html/template"
```
Comment (Luap99) [inline cmd/podman/inspect/inspect.go:6]: I think this should be `text/template`

Comment (jwhonce) [reply cmd/podman/inspect/inspect.go]: @Luap99 Good catch my IDE auto-imported the wrong package everywhere and I missed that one.

**Comment by ParkerVR** [comment]
looks good to me

**Comment by rhatdan** [comment]
@jwhonce Does this fix any of the table tests that we are currently skipping for remote?

ps_test.go:		Skip("FIXME: table still not supported in podman ps command")
volume_ls_test.go:		Skip("FIXME: table still not supported in podman volume command")

**Comment by rhatdan** [comment]
Does not fix
```
Podman ps
/home/dwalsh/go/src/github.com/containers/podman/test/e2e/ps_test.go:16
  podman ps namespace flag with go template format [It]
  /home/dwalsh/go/src/github.com/containers/podman/test/e2e/ps_test.go:220

  Expected
      <bool>: false
  to be true

  /home/dwalsh/go/src/github.com/containers/podman/test/e2e/ps_test.go:227
```
Does not fix
```
Podman volume ls
/home/dwalsh/go/src/github.com/containers/podman/test/e2e/volume_ls_test.go:12
  podman ls volume with Go template [It]
  /home/dwalsh/go/src/github.com/containers/podman/test/e2e/volume_ls_test.go:58

  Expected
      <int>: 1
  to equal
      <int>: 2

  /home/dwalsh/go/src/github.com/containers/podman/test/e2e/volume_ls_test.go:66
```

**Comment by rhatdan** [comment]
Of course you did not make this change to them.

**Comment by jwhonce** [comment]
@rhatdan We have the list we're working through and making smaller PR's to aid in testing and reviews. Those commands are coming.

**Pull request merged**